dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -1438,10 +1439,16 @@ public Response receiveChecksumValidationResults(@PathParam("identifier") String
 
     @POST
     @Path("{id}/submitForReview")
-    public Response submitForReview(@PathParam("id") String idSupplied) {
+    public Response submitForReview(@PathParam("id") String idSupplied, String jsonBody) {
```

Comment:
jsonbody should already be an object in parameter. AFAIK jersey will take care of it, same way as spring does. So there should be dto with 1 parameter.

Reply:
I'm not sure what do you propose here. Changing `String` to `Json` wouldn't be consistent with current API

Reply:
I'm not sure what you mean, String is json in this case.

Reply:
Doesn't seem to work and exploring this is not in scope of this task. I'll leave it the way it is until we decide we want a better approach

Reply:
Well if you can't get it working then we can create another task for it, but every task has a scope of doing stuff properly/clean (make sure you also tried to configure it similar to https://mkyong.com/webservices/jax-rs/json-example-with-jersey-jackson/ , we have config in ApiConfiguration class though). But you also can't leave it like this, we talked that alternative approach would be to use Jackson mapper method to map json.